frontend/src/components/KebabMenu/KebabMenu.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import "./index.less";
+
+const KebabMenuIcon = () => (
```

Comment:
I just made our own because I didnt find it after looking for `kebab` or under the `menu` section. Since `MoreOutlined` exists, this component is no longer needed and will be removed.